code-of-conduct.md

```diff
@@ -0,0 +1,70 @@
+# Code of Conduct
+
+## Summary
+We value the involvement of everyone in the community. We are committed to creating a friendly and respectful place for learning, teaching, and contributing. All participants in our events and communications are expected to show respect and courtesy to others.
```

Comment:
Line breaks here please. Both between the header and the beginning of the text `We value` and before the new sentence `All participants`.